missioncontrol/openapi/openapi.yaml

```diff
@@ -1172,26 +1172,24 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/Error'
-  /files/get_post_data_fields/:
-    get:
+  /files/presign/:
+    post:
```

Comment:
Yeah I was debating this one, it's certainly nicer to be able to schema it, but given it doesn't actually save anything to the DB I wasn't sure.